src/Blocks/BlockBigFlower.h

```diff
@@ -28,24 +28,31 @@ class cBlockBigFlowerHandler :
 		NIBBLETYPE Meta = a_ChunkInterface.GetBlockMeta(a_BlockX, a_BlockY, a_BlockZ);
 		if (Meta & 0x8)
 		{
-			super::DropBlock(a_ChunkInterface, a_WorldInterface, a_BlockPluginInterface, a_Digger, a_BlockX, a_BlockY - 1, a_BlockZ, a_CanDrop);
+			if (a_ChunkInterface.GetBlock(a_BlockX, a_BlockY - 1, a_BlockZ) == E_BLOCK_BIG_FLOWER)
```

Comment:
You need to check if `a_BlockY - 1` is inside the world height (`cChunkDef::IsValidHeight()`)